app/src/main/java/com/duckduckgo/app/launch/LaunchBridgeActivity.kt

```diff
@@ -42,6 +42,7 @@ class LaunchBridgeActivity : DuckDuckGoActivity() {
 
         configureObservers()
 
+        // determineViewToShow() does things with livedata, can't be on IO dispatcher
         lifecycleScope.launch { viewModel.determineViewToShow() }
```

Comment:
Two thoughts about this:

1. `viewModel.determineViewToShow()` is `suspend`, so it could handle threading requirements internally meaning this code that calls it shouldn't have to be concerned about how it is called
2. if not wanting to touch `determineViewToShow()` to make 1) true, should we at least be explicit in saying `lifecycleScope.launch(dispatchers.main())` if that is what is needed. i know that functionally it's the same as `lifecycleScope.launch {}` since that defaults to main, but I also dislike it because you need to remember that's the case and would prefer us to be more explicit to help readability if we can.